java/memory/src/test/java/org/apache/arrow/memory/TestBaseAllocator.java

```diff
@@ -478,6 +520,79 @@ public void testAllocator_overAllocateParent() throws Exception {
     }
   }
 
+  @Test
+  public void testAllocator_parentLimit_outcomeDetails() throws Exception {
```

Comment:
it might be clearer to call this something like failureAtParentLimit

Reply:
renamed to failureAtParentLimitOutcomeDetails